string nameArm
string name
---
float64 angle
